### ISSUE TITLE
Bug: ProtonMail services shows incorrect unread messages

### ISSUE THREAD

**Issue by aravindhp** [ferdium/ferdium-recipes#263]
### Avoid duplicates

- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues) for a bug report similar to mine, in vain

### Ferdium Version

6.2.2

### What Operating System are you using?

Other Linux

### Operating System Version

Fedora 37

### What arch are you using?

x64

### Expected Behavior

The ProtonMail services should the correct number of unread message count in the inbox on the service icon

### Actual Behavior

Shows arbitrary number of unread message count in the inbox on the service icon. I have zero unread messages and it shows 20. I also have more than 20 unread messages in other folders and labels so not sure where the 20 is coming from.

### Steps to reproduce

1. Add the ProtonMail service
2. Check the icon service tab and you will see incorrect number of messages. 

### Screenshots

![image](https://user-images.githubusercontent.com/11638995/206585529-b5d13354-9208-4f7d-95dd-b9a255acf12d.png)

**Comment by cino** [comment]
Q: Do you have unread messages in spam? For me personally that's a reason for the number to be "incorrect"

**Comment by Latschenharry** [comment]
For me its working as expected. The unread messages counter is for all mails in every folder including the Spam folder

**Comment by aravindhp** [comment]
So today when I opened up Ferdium it again showed 20 unread messages. However, as I traversed from the inbox to another folder the count go reset to all mails in every folder including Spam. But is this expected behavior? Shouldn't unread messages reflect only unread messages in the inbox? Otherwise you are always going to have the unread messages indicator on the Ferdium icon.

**Comment by aravindhp** [comment]
More this behavior is not consistent with other services. In Gmail for example, the unread messages counter only reflect the inbox.

**Comment by DominusExult** [comment]
> Moreover this behavior is not consistent with other services. In Gmail for example, the unread messages counter only reflect the inbox.

yes, this was an issue that I reported in the ferdi days and it was shot down. Apparently people like their unread badges to reflect inbox+archive+spam for reasons I don't understand (and no other mail program or ferdium recipe does it like that).
Someone (Mattia Panzeri) helped me out and created this patch that makes it work correctly.
[protonunredcounter.diff.txt](https://github.com/ferdium/ferdium-recipes/files/10331373/protonunredcounter.diff.txt)

**Comment by aravindhp** [comment]
@DominusExult thanks, that did the trick. Any reason not to try again with Ferdium? Maybe it's maintainers will accept it.

**Comment by DominusExult** [comment]
no real reason just not ready for the possible rejection. 
No offense intended to both Ferdi and Ferdium devs - I understand different preferences. And I also know how difficult it is to deal with people in open source projects.
